frontend/src/components/3.Preferences/AutocompleteSearchBox.js

```diff
@@ -41,7 +50,7 @@ const AutocompleteSearchBox = ({ setLocation, sendCoordinates }) => {
             <input
               className={'SearchBox'}
               {...getInputProps({
-                placeholder: 'Grafton, Auckland, New Zealand',
+                placeholder:' Street, City, Country',
```

Comment:
Could you please remove the space before Street. My latest changes now adds padding to the left of the input text :)

Reply:
Ok changed, thanks for that Reeve